src/diffusers/schedulers/scheduling_consistency_models.py

```diff
@@ -220,7 +220,7 @@ def set_timesteps(
             self.num_inference_steps = num_inference_steps
 
             step_ratio = self.config.num_train_timesteps // self.num_inference_steps
-            timesteps = (np.arange(0, num_inference_steps) * step_ratio).round()[::-1].copy().astype(np.int64)
+            timesteps = (np.arange(0, num_inference_steps) * step_ratio).round()[::-1].copy().astype(np.int32)
```

Comment:
Won't it cause any numerical problems or qualitative problems?

Reply:
Should be ok but I'll cherry-pick this + the other fix (https://github.com/huggingface/diffusers/pull/10442/commits/64553b36ef2fdd65d9eec9c25a8a22ddf2d7eb76) and review with @yiyixuxu

Reply:
PR is now only workflow changes, we may want to just disable it until we get the new runner though.

Reply:
That is okay with me.